src/commands/utils/events.ts

```diff
@@ -0,0 +1,90 @@
+import fetch from 'node-fetch';
+import { Colors, EmbedField } from 'discord.js';
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import Logger from '../../lib/logger';
+import { makeEmbed } from '../../lib/embed';
+
+const BASE_VATSIM_URL = 'https://my.vatsim.net';
+
+const handleLocaleTimeString = (date: Date) => date.toLocaleTimeString('en-US', {
+    hour: 'numeric',
+    minute: 'numeric',
+});
+
+const handleLocaleDateString = (date: Date) => date.toLocaleDateString('en-US', {
+    weekday: 'short',
+    month: 'short',
+    day: 'numeric',
+});
+
+export const events: CommandDefinition = {
+    name: ['events', 'event', 'ev'],
```

Comment:
```suggestion
    name: ['events', 'event', 'ev', 'vatevents', 'vatsimevents'],
```

Just some suggestions to make aliases more specific. (Any you choose need adding to command-docs)